community-operators/federation/federation.v0.0.7.clusterserviceversion.yaml

```diff
@@ -250,21 +250,24 @@ spec:
 
     Use the `kubefed2 disable` command to disable federation for an API type.
 
-        kubefed2 disable deployments --federation-namespace=<namespace>
+        kubefed2 disable deployments.extensions --federation-namespace=<namespace>
 
-    This command deletes the `FederatedTypeConfig` for the API type being disabled.
+    This will disable the propagation of federated deployments, but leave the
+    API surface and `FederatedTypeConfig` intact.
 
-        kubectl get federatedtypeconfigs -n <namespace>
+    To fully remove the API surface, use the `--delete-from-api` flag:
+
+        kubefed2 disable deployments.extensions --federation-namespace=<namespace>
```

Comment:
kubefed2 disable deployments.extensions --federation-namespace=<namespace>  -delete-from-api

Reply:
Nice catch - I thought that I had fixed that. Thanks!